lib/Plans.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+// Copyright 2022. Plesk International GmbH. All rights reserved.
+
+namespace WHMCS\Module\Server\Plesk360Monitoring;
+
+use Exception;
+use WHMCS\Module\Server\Plesk360Monitoring\Plans\BusinessPlan;
+use WHMCS\Module\Server\Plesk360Monitoring\Plans\EnterprisePlan;
+use WHMCS\Module\Server\Plesk360Monitoring\Plans\FlexiblePlan;
+use WHMCS\Module\Server\Plesk360Monitoring\Plans\ProPlan;
+
+class Plans
+{
+    /**
+     * @return Plan[]
+     */
+    public static function getAll(): array
+    {
+        return [
+            new ProPlan(),
+            new BusinessPlan(),
+            new EnterprisePlan(),
+            new FlexiblePlan(),
+        ];
+    }
+
+    public static function getPlanById(string $id): Plan
+    {
+        foreach (self::getAll() as $plan) {
+            if ($plan->id() === $id) {
+                return $plan;
+            }
+        }
+
+        throw new Exception("Plan with id '{$id}' not found");
```

Comment:
`RuntimeException` would be the correct exception type because it's can only be found on runtime.
Even better would be a explicit exception

Reply:
Fixed